src/science/canopy/cbl_init_wetfac_mod.f90

```diff
@@ -0,0 +1,103 @@
+MODULE cable_init_wetfac_mod
```

Comment:
I'm not keen on creating a separate file and module for this single subroutine. 
@JhanSrbinovsky couldn't we move that within the `cbl_SurfaceWetness.F90` file and module? I know it means applying the same change in ACCESS3-JULES but that isn't insurmountable. This "init" module isn't part of the code initialisation but is within the science code, so having it within the science module it pertains to seems better to me.

Reply:
Let's not to this now.